### ISSUE TITLE
[rocprofiler] CMake variables `NOTFOUND`

### ISSUE THREAD

**Issue by tpkessler** [rocm-arch/rocm-arch#47]
This issue is similar to #46 
Again, `rocm` is listed as dependency. When removed, the configuration step fails:

```bash
---------Install-Dir: /opt/rocm/rocprofiler/rocprofiler
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
HSA_KMT_LIB
    linked by target "ctrl" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
    linked by target "standalone_test" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
    linked by target "intercept_test" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
HSA_RUNTIME_LIB
    linked by target "rocprofiler64" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0
    linked by target "tool" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
    linked by target "ctrl" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
    linked by target "standalone_test" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test
    linked by target "intercept_test" in directory /home/torsten/pkg/rocm-arch/rocprofiler/src/rocprofiler-roc-3.1.0/test

-- Configuring incomplete, errors occurred!
```

**Comment by acxz** [comment]
`rocm` is provided by `rocm-dkms`.

**Comment by tpkessler** [comment]
The problem persists, see #46

**Comment by acxz** [comment]
I can't reproduce it as of yet, I get the following issue: https://aur.archlinux.org/packages/rocm-clang-ocl/#comment-733764

**Comment by tpkessler** [comment]
I fixed the build script, see my pr #61 
This also answers issue #49.
